CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `Progress` component
```

Comment:
Think we need to rename this component since we already got a `ProgressBar` component.

Reply:
yes, I will do this and maintain his compatibility with this one.